src/data/Pet.js

```diff
@@ -39,7 +49,7 @@ export class Pet {
      * Returns true if the pet needs updated.
      */
     updatePet(){
-        if (this.poopCooldown()){   //Can just add methods to this if statement
+        if (this.poopCooldown() || this.thoughtCooldown()){   //Can just add methods to this if statement
```

Comment:
It feels a bit redundant to me to check the value of this conditional just to return what it evaluates to in both cases, when we could just return the evaluation of the condition itself (`return (this.poopCooldown() || this.thoughtCooldown())`).

It's really not a problem though and changing it would do practically nothing other than make this already small method slightly less verbose

Reply:
no that's a good point I'll do that